Sources/SwiftDocC/Utility/FeatureFlags.swift

```diff
@@ -40,4 +40,11 @@ public struct FeatureFlags: Codable {
         additionalFlags: [String : Bool] = [:]
     ) {
     }
+
+    /// Set feature flags that were loaded from a bundle's Info.plist.
+    public mutating func loadFlagsFromBundle(_ bundleFlags: DocumentationBundle.Info.BundleFeatureFlags) {
```

Comment:
Small nit: should we move this code to FeatureFlags+Info.swift so that it's all in the same spot?

Reply:
I think it could go either way. I put it here so that people who add feature flags in the future could see where they're being loaded from the Info.plist, and decide to add it there if desired. However, writing this method alongside the `BundleFeatureFlags` definition means that the code that defines the feature flag and the code that applies it to the global feature flags lives in the same place.

I think the major reason i wrote it this way is because when i started writing this PR, i had a hard time figuring out where the Info.plist data was even defined! 😅 But i can move it if the others also feel like it should live by `BundleFeatureFlags`.